src/server/rdb_save.cc

```diff
@@ -654,6 +665,9 @@ std::error_code RdbSerializer::SaveSBFObject(const PrimeValue& pv) {
 
     string_view blob = sbf->data(i);
     RETURN_ON_ERR(SaveString(blob));
+    if (i + 1 == sbf->num_filters()) {
+      is_last_entry_ = true;
```

Comment:
There is also `ZSET` which I haven't modified. The problem there is that it uses `Iterate()` and a `callback` but within the callback we don't really know if we are on the last `value` and therefore we can't really know if we are serializing the last entry of the `key/value`. I could change the `Iterate` interface but I find that intrusive. Any suggestions ?

Reply:
if you know the size you could count maybe?